ports/mimxrt/Makefile

```diff
@@ -101,12 +101,15 @@ SRC_TINYUSB_IMX_C += \
 
 SRC_C = \
 	main.c \
+	adc.c \
 	led.c \
 	pin.c \
 	ticks.c \
 	tusb_port.c \
 	board_init.c \
 	$(BOARD_DIR)/flash_config.c \
+	$(MCU_DIR)/drivers/fsl_adc.c \
```

Comment:
please place this line in the `SRC_TINYUSB_IMX_C` macro above (which is wrongly named, but at least it's the right place)

Reply:
I changed the name of the macro to `SRC_HAL_IMX_C` and added the respective line.